tests/puppeteer/campaigns/functional/BO/catalog/categories/01_filterAndQuickEditCategories.js

```diff
@@ -8,7 +8,7 @@ const {Categories} = require('@data/demo/categories');
 const BOBasePage = require('@pages/BO/BObasePage');
 const LoginPage = require('@pages/BO/login');
 const DashboardPage = require('@pages/BO/dashboard');
-const CategoriesPage = require('@pages/BO/categories');
+const CategoriesPage = require('@pages/BO/catalog/categories/categories');
```

Comment:
Redundant. If categories is a directory, you should be able to create and index.js file in this one
```suggestion
const CategoriesPage = require('@pages/BO/catalog/categories');
```